far/DlgBuilder.hpp

```diff
@@ -1,4 +1,4 @@
-﻿#ifndef DLGBUILDER_HPP_E8B6F5CA_37A9_403A_A3F0_9ED7271B2BA7
+#ifndef DLGBUILDER_HPP_E8B6F5CA_37A9_403A_A3F0_9ED7271B2BA7
```

Comment:
What's been changed here?

Reply:
Sorry about that. Accidentally lost BOM. Fixed.

Reply:
Ok, squash please.

Reply:
Done

Reply:
Thanks